src/ProjectFileIO.cpp

```diff
@@ -283,6 +284,24 @@ ProjectFileIO::ProjectFileIO(AudacityProject &project)
    mTemporary = true;
 
    UpdatePrefs();
+
+   // Make sure there is plenty of space for Sqlite files
+   wxLongLong freeSpace = 0;
```

Comment:
I'm not sure why this change is a part of this pull request.

Why 100 Mb? Why 100 mb is written that way? 100 * 1024 * 1024 expresses it better.

Why `AbbreviatePath` is used? It is confusing on non Windows platforms and not entirely correct on Windows as well. NTFS is much more complicated than just drive letters.

Also I have strong opinion that disk space/writability errors should allow users to easily change the problematic location, which is already implemeted in Audacity anyway. The current message is unhelpfull, confusing and sometimes even wrong.

Reply:
Ok, I see now why it is a part of the request, I don't agree with implementation still. We should open an issue about improving this error.

Reply:
You understand the need to cut and paste the code that existed to some other place to break dependency.  I'm not changing the implementation.